src/utils/error-message/error-message.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
Tu akurat masz 2 wybory: Albo robisz tak jak żąda eslint - czyli dajemy NOT_FOUND, albo zmieniasz to na obiekt z wiadomościami. To też jest opcja, by np. ustalić taki schemat - jak wiadomości, to robimy to jako obiekt, a nie enum (obiekt byłby nawet lepszy).